### ISSUE TITLE
Episodes in "Child Series" missing from RSS Feed

### ISSUE THREAD

**Issue by KTorrii** [CastosHQ/Seriously-Simple-Podcasting#691]
**Describe the bug**
The episodes assigned to a Child Series are missing from the child-series RSS Feed.

**To Reproduce**
1. Go to or create a Child Series under a Parent Series
2. Assign/Verify episodes are assigned to that child series.
3. View the RSS Feed for the child series
4. See error: No episodes shown.

**Expected behavior**
The episodes assigned to the child series should be displayed in the child series RSS Feed. 

**Screenshots**
![image](https://user-images.githubusercontent.com/76892425/117384560-45e4d080-aeb1-11eb-8adb-17873c42811b.png)

![image](https://user-images.githubusercontent.com/76892425/117384613-614fdb80-aeb1-11eb-8b7c-8240dec8f747.png)

**Comment by zahardev** [comment]
@TheCraigHewitt Should we support the hierarchy for the series?

**Comment by TheCraigHewitt** [comment]
@zahardev yes i think we should.  So if an episode is assigned to a child series then it should appear in that child series' RSS feed.
